persistence/module.go

```diff
@@ -64,6 +75,18 @@ func Create(cfg *genesis.Config, genesis *genesis.GenesisState) (modules.Persist
 	return persistenceMod, nil
 }
 
+func InitGenesis(data json.RawMessage) (genesis *types.PersistenceGenesisState, err error) {
```

Comment:
As discussed in protocol hour, let's enforce `InitGenesis` and `InitiConfig` as part of the module interface.

We currently have this:

```
type Module interface {
	IntegratableModule
	InterruptableModule
}

type IntegratableModule interface {
	SetBus(Bus)
	GetBus() Bus
}

type InterruptableModule interface {
	Start() error
	Stop() error
}
```

We could a `ConfigurableModule` to the mix